app/code/Magento/UrlRewriteGraphQl/composer.json

```diff
@@ -5,7 +5,10 @@
     "require": {
         "php": "~7.3.0||~7.4.0",
         "magento/framework": "*",
-        "magento/module-url-rewrite": "*"
+        "magento/module-url-rewrite": "*",
+        "magento/module-catalog": "*",
```

Comment:
we can't have here:
magento/module-catalog”: “*”,
        “magento/module-cms-graph-ql”: “*”,
        “magento/module-catalog-graph-ql”: “*”